web/src/components/config_render/ConfigItemTitle.jsx

```diff
@@ -24,10 +24,10 @@ export default class ConfigItemTitle extends React.Component {
     return (
       <h4
         className="card-item-title field-section-sub-header"
-        style={title ? { marginBottom: -18 } : {}}
+        style={title?.length < 101 ? { marginBottom: -18 } : {}}
```

Comment:
No marginBottom if we have title length of more than 101. The "required/recommended" tags will overlap with title if theres a negative margin.